components/homme/src/share/gllfvremap_util_mod.F90

```diff
@@ -168,14 +208,29 @@ subroutine set_gll_state(hvcoord, elem, nt1, nt2)
     end do
   end subroutine set_gll_state
 
+  function make_tendency(p) result(t)
```

Comment:
Is this a forcing function tied to the special Cinf IC? If not, what is this for?

Reply:
This is for gllfvremap internal functional property unit tests. I suggest ignoring this.